index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>Skeleton project</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+
+  <body>
+    <header>
+      <section class="navigator">
+          <a href="#" class="mylogo">My logo</a>
+          <nav>
+            <img src="images/Enabled.png" alt="" class="hamburger">
+          </nav>
+      </section>
+    </header>
+
+    <main class="container">
+        <h1 class="head">I’m Suzana <br> Glad to see you!</h1>
+        <p class="pr">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <a href="#" class="a">LET’S CONNECT</a>
```

Comment:
- [ ] As the previous reviewer suggested, let's remove the `a` tag around the "LET'S CONNECT" since it's not going to be linking to anything. It is just there to serve as a call to action so the user knows to click on the icons below which are the actual links. You can use any of the `hx`(headings) but not the `h1` tag